ckanext/validation/jobs.py

```diff
@@ -120,7 +120,7 @@ def run_validation_job(resource):
 
 def _validate_table(source, _format=u'csv', schema=None, **options):
 
-    http_session = requests.Session()
+    http_session = options.pop('http_session', None) or requests.Session()
```

Comment:
do we have a test for this, or can you create one. Also need a bit more info as to what magic or doc you read that suggests this.

Reply:
I didn't read any docs about it, but on some CKAN versions, there was a duplicate key error when adding 'http_session', indicating that the session already existed, so I added this to detect that and reuse it if applicable.